extensions/ql-vscode/test/vscode-tests/cli-integration/data-extensions-editor/modeled-method-fs.test.ts

```diff
@@ -52,6 +52,14 @@ describe("modeled-method-fs", () => {
   let workspacePath: string;
   let cli: CodeQLCliServer;
 
+  beforeAll(async () => {
+    const extension = await getActivatedExtension();
+    cli = extension.cliServer;
+
+    // The java-all pack needs to be available for codeql resolve extensions to succeed.
```

Comment:
Just checking, it's not something special about the java language, but it's because we happened to pick java for our tests?